network-store-iidm-impl/src/main/java/com/powsybl/network/store/iidm/impl/NodeBreakerViewImpl.java

```diff
@@ -131,6 +135,25 @@ public void traverse(int node, Traverser traverser) {
         traverse(graph, node, traverser, done);
     }
 
+    void traverse(Terminal terminal, VoltageLevel.TopologyTraverser traverser, Set<Terminal> traversedTerminals) {
```

Comment:
I don't think this is the right to implement this: most of the traversal should be done by tracking traversed nodes and not terminals (and in bus/breaker configured buses instead of terminal). I am also wondering if it really works using Set<Terminal> as Terminals are not stable by identity.

Reply:
Remove the use of the `isConnected` method for a terminal in node breaker topology traversal